src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -21,6 +22,8 @@
 
     private transient boolean needCheckStyle = false;
     private transient int commitNum = 1;
+    private transient List<String> whiteListedFileTypes = Arrays.asList(".java", ".adoc", ".js", ".md", ".css",
+            ".html", ".cs", ".json", ".xml", ".py", ".fxml", ".tag", ".jsp", ".gradle");
```

Comment:
I'm thinking whether it would be better to configure this at the main because once you integrate it with the parser/config, it would be clearer to state it like `load param if exist else load default` rather than overwrite it silently at the backend.

Reply:
@yong24s is already integrating this into parser in #159, will put this in `main` temporary in this PR for v1.1